docs-v2/vercel.json

```diff
@@ -1,14 +1,4 @@
 {
-  "rewrites": [
```

Comment:
You'll want to move these to the `redirects` section instead of removing them entirely, since we still have some `/docs/v3` links out there, so we want to redirect `/docs/v3` to `/docs`